iphone/Classes/TiLogServer.m

```diff
@@ -283,6 +319,13 @@ +(void)startServer
 		[connections addObject:conn];
 		forcedNSLog(@"[INFO] Log server connections: %d", [connections count]);
 
+		// send the header
+		dispatch_data_t buffer = dispatch_data_create(headers.bytes,
+													  headers.length,
```

Comment:
That's the Xcode default. I don't love it. Other than putting everything on a single line, how would you like to see it indented?